compiler/src/main/kotlin/io/spine/protodata/renderer/SourceFile.kt

```diff
@@ -145,6 +157,14 @@ private constructor(
         }
     }
 
+    /**
+     * Deletes this source file from the file system.
+     */
+    internal fun rm(rootDir: Path) {
```

Comment:
Please document the parameter. It looks confusing, given that _this_ file only resides in a single place on the current-generation FS`es.